OBAKit/Controls/ListView/OBAContentView.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  OBAContentView.swift
+//  OBAKit
+//
+//  Created by Alan Chu on 10/8/20.
+//
+
+import Foundation
+
+/// The requirements for a content view that you create using a configuration.
```

Comment:
Can you elaborate on where an `OBAContentView`-conforming object would be used? Doesn't have to be a ton of detail, but say something like "these content views are used in our list views for blah blah blah"